user_templates/mcu_defaults/main/BTT_SKR_Pico_v1.0.cfg

```diff
@@ -0,0 +1,34 @@
+
+#----------------------------------------#
+#### BTT SKR Pico v1.0 definition ########
+#----------------------------------------#
+
+[mcu]
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+##--------------------------------------------------------------------
+
+[include config/mcu_definitions/main/BTT_SKR_Pico_v1.0.cfg] # Do not remove this line
+[board_pins xye_SKR_mcu]
+mcu: mcu
+aliases:
+    X_STEP=MCU_XM_STEP , X_DIR=MCU_XM_DIR , X_ENABLE=MCU_XM_ENABLE ,
+    Y_STEP=MCU_YM_STEP , Y_DIR=MCU_YM_DIR , Y_ENABLE=MCU_YM_ENABLE ,
+    Z_STEP=MCU_ZM_STEP , Z_DIR=MCU_ZM_DIR , Z_ENABLE=MCU_ZM_ENABLE ,
+    E_STEP=MCU_EM_STEP , E_DIR=MCU_EM_DIR , E_ENABLE=MCU_EM_ENABLE ,
+
+    XYZE_TMCUART=MCU_TMCUART , XYZE_TMCTX=MCU_TMCTX , 
```

Comment:
Can you change this to `TMCUART` and `TMCTX` directly? I think there is no need for the XYZE in front

Reply:
i added it to make it coherent to understand for which axis it is. if you put nothing in front of it, it is not as clear as when having it in front, imo

Reply:
Yeah but TMC are only used for axis movements and I don't know if there is any board that use multiple UART for groups of TMC. It's always one UART for all TMC (as for this board) or multiple UART with one per TMC (like for the other boards supported by Klippain)